templates/admin/certification_request/show.html.twig

```diff
@@ -184,6 +184,42 @@
                             <p class="form-text text-muted text-center">
                                 {{ macros.get_certification_request_processed_by(object) }}
                             </p>
+
+                            {% if object.blockReason %}
+                                <p class="form-text text-muted text-center">
+                                    Bloquée:
+                                    <b>
+                                        {% if constant('App\\Adherent\\CertificationRequestBlockCommand::BLOCK_REASON_OTHER') == object.blockReason %}
+                                            {{ object.customBlockReason }}
+                                        {% else %}
+                                            {{ ('certification_request.block_reason.' ~ object.blockReason)|trans }}
+                                        {% endif %}
+                                    </b>
+
+                                    {% if object.blockComment %}
+                                        <br>
+                                        ({{ object.blockComment }})
+                                    {% endif %}
+                                </p>
+                            {% endif %}
+
+                            {% if object.refusalReason %}
+                                <p class="form-text text-muted text-center">
+                                    Refusée:
+                                    <b>
+                                        {% if constant('App\\Adherent\\CertificationRequestRefuseCommand::REFUSAL_REASON_OTHER') == object.refusalReason %}
```

Comment:
as an alternative for avoiding to do this check many times: `object.refusedWithOtherRaison(): bool` helper